packages/shared/lib/core/wallet/utils/getActivityTileTitle.ts

```diff
@@ -28,8 +28,8 @@ export function getActivityTileTitle(activity: Activity): string {
     } else if (activity.type === ActivityType.Consolidation) {
         return isConfirmed ? 'general.consolidated' : 'general.consolidating'
     } else if (action === ActivityAction.Mint) {
```

Comment:
Currently, the action of creating an account  is not detected as `mint`.
It would also be necessary to create the case for when an account output is sent and its respective literals.

Reply:
I've opened an [issue](https://github.com/iotaledger/firefly/issues/8021) to resolve this :pray: